core/src/main/kotlin/org/eln2/sim/thermal/main.kt

```diff
@@ -0,0 +1,146 @@
+import kotlinx.coroutines.*
+import kotlinx.coroutines.sync.*
+import kotlin.math.*
+
+class ThermalInterface{
```

Comment:
Little things: it looks better to leave a space around most punctuation, including braces, separating it from alphanumerics.